addons/sms/static/tests/sms_test_helpers.js

```diff
@@ -0,0 +1,10 @@
+
+import { busModels } from "@bus/../tests/bus_test_helpers";
+import { mailModels } from "@mail/../tests/mail_test_helpers";
+import { Partner } from "@sms/../tests/mock_modals/partner";
+import { Visitor } from "@sms/../tests/mock_modals/visitor";
+import { defineModels, webModels } from "@web/../tests/web_test_helpers";
+
+export function defineSMSModels() {
+    return defineModels({ ...busModels, ...webModels ,...mailModels, Partner, Visitor });
```

Comment:
This should export a `smsModel` variable an use it inside the define models (Exemple https://github.com/odoo/odoo/pull/164876/files#diff-2dfe6242e2341501f821305327bd2dc35b92ca1e05a6b37abcea268953b8b40aR10)